### ISSUE TITLE
The source code button is missing an alternative text that describes its function

### ISSUE THREAD

**Issue by rautenburger** [swiss/styleguide#565]
The source code button needs an alternative text that describes its function. The visual impaired user has no indication what happens after pushing this button. 

Tested on:
https://q-src.github.io/styleguide/en/
(Chrome - Version 58.0.3029.110)
Tested with:
NVDA screen reader

**Comment by bit-pro-iew-eui** [comment]
The source code button "<>" everywhere in the guideline

**Comment by janissbinder** [comment]
It will not affect anything if I add an alternativ text to the source code button. The alt attribute can only be used with <input type="image"> (Source: https://www.w3schools.com/TAGS/att_input_alt.asp). Please let me know if you still want this.
